TAGradingServer/account/submit/admin-classlist.php

```diff
@@ -152,29 +156,36 @@ function($csv_file, $xlsx_file) {
 // student_manual is true)
 \lib\Database::beginTransaction();
 foreach ($rows as $row) {
-    // TODO add email
-	$columns = array("user_id", "user_firstname", "user_lastname", "user_email", "registration_section");
-	$values = array($row['student_user_id'], $row['student_first_name'], $row['student_last_name'], $row['student_section'], 1);
-	$user_id = $row['student_user_id'];
+	$columns = array("user_id", "user_firstname", "user_lastname", "user_email", "user_group","registration_section");
+	$values = array($row['user_id'], $row['student_first_name'], $row['student_last_name'], $row['student_email'],4, $row['registration_section']);
+	$user_id = $row['user_id'];
 	if (array_key_exists($user_id, $students)) {
-		if (isset($_POST['ignore_manual_1']) && $_POST['ignore_manual_1'] == true && $students[$user_id]['student_manual'] == 1) {
+//<<<<<<< HEAD
+		if (isset($_SESSION['post']['ignore_manual_1']) && $_SESSION['post']['ignore_manual_1'] == true && $students[$user_id]['manual_registration'] == 1) {
+//=======
+	//	if (isset($_POST['ignore_manual_1']) && $_POST['ignore_manual_1'] == true && $students[$user_id]['student_manual'] == 1) {
+//>>>>>>> origin/master
```

Comment:
We want to use master here. Because we're not doing a redirect chain, ```$_SESSION['post']``` will never be set and we can always just use ```$_POST```